metrique/src/lib.rs

```diff
@@ -405,3 +405,6 @@ impl<M: InflectableEntry> Entry for RootEntry<M> {
 pub use metrique_service_metrics::ServiceMetrics;
 
 pub use metrique_core::concat;
+
+// re-export FormatDisplay
```

Comment:
what do you think about putting this in a `format` module? The top-level is getting really busy and given the very generic name `ToString`, a format module might make it a little clearer.